test/WidenImage.js

```diff
@@ -0,0 +1,78 @@
+import applib from '../build/applib'
+import assert from 'assert'
+import fixtureIO from './Utilities/FixtureIO'
+import styleMocking from './Utilities/StyleMocking'
+
+const maybeWidenImage = applib.WidenImage.maybeWidenImage
+let document = null
```

Comment:
For whatever reason, I never thought to use a top level variable in my PR. Any case for or against 1) top level variable vs 2) `this.document` in `beforeEach()`? I'm not sitting on anything informative but we should be consistent one way or another.